deploy/roles/citus/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Add Citus rpm
+  get_url:
+    url: "{{ rpm_url }}"
+    dest: /tmp/rpm.sh
+    mode: "0755"
+  tags:
+    - citus
+
+- name: Execute rpm script
+  become: yes
+  become_method: sudo
+  shell: ./rpm.sh
+  args:
+    chdir: /tmp/
+  when: not custom_repo
+  tags:
+    - citus
+
+- name: Install Citus
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ citus_pkg }}"
+  tags:
+    - citus
+
+- name: Add pgpass file
+  become: yes
+  become_user: "{{ pp_superaccount }}"
+  template:
+    src: pgpass.j2
+    dest: ~/.pgpass
+    mode: "0600"
+  tags:
+    - citus
```

Comment:
One thing I forgot to mention from the Design Document, we are planning to use CGroups to isolate Citus, which will allow for future multipurpose use of these servers.  This should be relatively straightforward thanks to SystemD on RHEL 7, here's the relevant docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/resource_management_guide/sec-Modifying_Control_Groups#sec-Modifying_Unit_Files.  I'm thinking we could have two variables, `citus_mem_limit` and `citus_cpu_limit`, which would be undefined by default, and the relevant tasks would have a clause such as: `when: citus_mem_limit is defined and citus_cpu_limit is defined`.  We could then test with `EXTRA_VARS='{citus_mem_limit: 1024, citus_cpu_limit: 50}'` or whatever is appropriate when using Vagrant, and define them in Production in group_vars.